gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
-|  direction_id | Enum | Optional | The **direction_id** field contains a binary value that indicates the direction of travel for a trip. Use this field to distinguish between bi-directional trips with the same **route_id**. This field is not used in routing; it provides a way to separate trips by direction when publishing time tables. You can specify names for each direction with the **trip_headsign** field. |
-|   |  | | * 0 - travel in one direction (e.g. outbound travel) |
-|   |  | | * 1 - travel in the opposite direction (e.g. inbound travel) |
-|   |  | | For example, you could use the *trip_headsign* and *direction_id* fields together to assign a name to travel in each direction for a set of trips. A [trips.txt](#tripstxt) file could contain these rows for use in time tables: |
-|   |  | | * `trip_id,...,trip_headsign,direction_id` |
-|   |  | | * `1234,...,Airport,0` |
-|   |  | | * `1505,...,Downtown,1` |
-|  block_id | ID | Optional | The **block_id** field identifies the block to which the trip belongs. A block consists of a single trip or many sequential trips made using the same vehicle, defined by shared service day and block_id. A block_id can have trips with different service days, making distinct blocks. (See [example below](#example-showing-blocks-and-service-day)) |
-|  shape_id | ID | Optional | The **shape_id** field contains an ID that defines a shape for the trip. This value is referenced from the [shapes.txt](#shapestxt) file. The shapes.txt file allows you to define how a line should be drawn on the map to represent a trip. |
-|  wheelchair_accessible | Enum | Optional | * **0** (or empty) - indicates that there is no accessibility information for the trip |
-|   |  | | * **1** - indicates that the vehicle being used on this particular trip can accommodate at least one rider in a wheelchair |
-|   |  | | * **2** - indicates that no riders in wheelchairs can be accommodated on this trip |
-|  bikes_allowed | Enum | Optional | 0 (or empty) - indicates that there is no bike information for the trip |
-|   |  | | * **1** - indicates that the vehicle being used on this particular trip can accommodate at least one bicycle |
-|   |  | | * **2** - indicates that no bicycles are allowed on this trip |
-
-#### Example showing blocks and service day
+|  route_id | ID referencing `routes.route_id` | **Required** | Identifies a route. |
+|  service_id | ID referencing `calendar.service_id` or `calendar_dates.service_id` | **Required** | Identifies a set of dates when service is available for one or more routes. |
+|  trip_id | ID | **Required** | Identifies a trip. |
+|  trip_headsign | Text | Optional | Signage text that identifies the trip's destination to riders. Use this field to distinguish between different patterns of service on the same route. If the headsign changes during a trip, `trip_headsign` can be overridden by specifying values for the `stop_times.stop_headsign`. |
```

Comment:
Subtle change:
"Text that appears on signage identifying the trip's destination to riders."

(I don't know if "signage text" would be universally clear.)